incubator/hnc/internal/reconcilers/anchor.go

```diff
@@ -118,13 +119,19 @@ func (r *AnchorReconciler) onDeleting(ctx context.Context, log logr.Logger, inst
 		return false, err
 	}
 
+	// Update the state of the anchor
+	//
+	// TODO: call this in the parent function after v0.5 so there's no special code path for the
```

Comment:
Is there already an issue created for this?

Reply:
I'm going to do it as I cherrypick this change into master, starting ~now :)

Reply:
Update: fixed in #1151